department-of-veterans-affairs/caseflow
# Assign Hearings Skeleton

Resolves #6916 

### Description
This PR adds a skeleton page at /hearings/schedule/assign.

### Testing Plan
1. Go to /hearings/schedule/assign as either 'Build HearSched' or 'Edit HearSched' and confirm the page loads
1. Go to /hearings/schedule/build as 'Build HearSched' and confirm the page loads
1. Go to /hearings/schedule/build as 'Edit HearSched' and confirm the page doesn't load
